site/en/docs/extensions/reference/storage/index.md

```diff
@@ -15,14 +15,22 @@ The Storage API provides an extension-specific way to persist user data and stat
 
 {% Details %}
 {% DetailsSummary %}
-💡 Can extensions use the WebStorage API?
+💡 Can extensions use the Storage API?
 {% endDetailsSummary %}
 
-Even though extensions can access [`Window.localStorage`][mdn-localstorage] in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+Even though extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
 
-- The extension's service worker cannot access `Window.localStorage`.
-- Content scripts share the `Window.localStorage` of the host page.
-- Data saved to `Window.localStorage` is lost when the user clears their browsing history.
+- Extension's service worker cannot access `Storage`.
+- Content scripts share storage with the host page.
+- Data saved using the `Storage` interface is lost when the user clears their browsing history.
+
+Even though extension service workers can use `chrome.storage`, if need to move data, from the `Storage` interface to `chrome.storage` you'll need to use an [offscreen document][offscreen-document]. 
```

Comment:
```suggestion
Extension service workers can use `chrome.storage`; however, moving data from the `Storage` interface to `chrome.storage` requires an [offscreen document][offscreen-document]. 
```

Reply:
Thinking about this some more, this is a reference page and not a migration one, so I'm actually not sure that we should even have something about offscreen documents - I think it just adds confusion. If we do want it though I think we need to be more explicit about why we're suggesting it:

```suggestion
Extension service workers can use `chrome.storage` and in most cases, this is preferable. If you previously utilized `Storage` in an MV2 extension, and need to migrate data, you can use an [offscreen document][offscreen-document]. 
```